test/mockengine.sh

```diff
@@ -36,6 +36,7 @@ runApp() {
 cd $BASE_DIR/../client-frameworks-support/testing-utilities/test
 #Run acutal test
 PORT=8181
+npm install
```

Comment:
Why the addition here. This is supposed to already have things installed in the previous build stage of the pipeline . I am getting the feeling this acts as some kind of pseudo Sleep function which keeps it from failing. If thats the case then need to find the root cause.

Reply:
Without this command "Mock Engine Tests" are failing